_config.yml

```diff
@@ -187,99 +167,85 @@ t:
       content: "Pour en savoir plus envoyez-nous un mail ou contactez-nous sur les réseaux sociaux. Nous vous répondrons le plus rapidement possible."
     medias:
       title: "Ils parlent de nous"
-     
+
     team:
       title: "L'équipe actuelle"
     partners:
       title: "Avec qui on travaille?"
       items:
-        - 
-          title: "SDIS 07"
+        - title: "SDIS 07"
           url: "https://www.sdis07.fr/"
           logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
           content: "Notre partenaire pour le recueil des besoins et l'expérimentation terrain"
           height: 75
-        - 
-          title: "Engie Green"
+        - title: "Engie Green"
           url: "https://www.engie-green.fr/"
           logo: "https://www.engie-green.fr/wp-content/themes/engie-green/assets/img/engie-green-logo.svg"
           content: "Notre premier partenaire privé, avec qui on réalise une expérimentation en gironde"
           height: 80
-        -
-          title: "CENIA"
+        - title: "CENIA"
           url: "https://cenia.cl/"
           logo: "/img/cenia.jpg"
           content: "Le centre national d'intelligence artificielle chilien avec qui on travaille pour notre premier déploiement à l’étranger"
           height: 60
-        - 
-          title: "ADF Subirats"
+        - title: "ADF Subirats"
           url: "https://subirats.cat/adfsubirats/"
           logo: "https://i.postimg.cc/NfW4Xyx7/camera-logo.png"
           content: "Notre partenaire catalan pour l’adaptation de notre dispositif à des caméras existantes"
           height: 80
-        - 
-          title: "FORCE 06"
+        - title: "FORCE 06"
           url: "https://www.departement06.fr/preservation-du-cadre-de-vie/force-06-3080.html"
           logo: "/img/force06_logo.png"
           content: "Notre partenaire des alpes maritimes afin de diversifier les tests terrains"
           height: 80
-        -
-          title: "SDIS 12"
+        - title: "SDIS 12"
           url: "https://www.sdis12.fr/"
           logo: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSg2bUbkvsTyiTM7uHqbHBY0nxzu7L9AGHEGg&s"
           content: "Notre partenaire dans l'Aveyron afin de diversifier les tests terrains"
           height: 125
-        -
-          title: "SDIS 77"
+        - title: "SDIS 77"
           url: "https://www.sdis77.fr/"
           logo: "https://www.infirmiersapeurpompier.com/wp-content/uploads/2020/11/77.jpg"
```

Comment:
The new logo is missing in french translation, could you point to the new one as in english ? thks